src/Compilers/CSharp/Test/Symbol/Symbols/Retargeting/RetargetingTests.cs

```diff
@@ -120,6 +121,39 @@ class C
             RetargetingSymbolChecker.CheckSymbols(sourceNamespace.GetMember<NamedTypeSymbol>("C"), retargetingNamespace.GetMember<NamedTypeSymbol>("C"));
         }
 
+        [Fact, CompilerTrait(CompilerFeature.InitOnlySetters)]
+        public void RetargetProperties_WithInitOnlySetter()
+        {
+            var source =
+@"interface I
+{
+    object Property { get; init; }
+}";
+            var compilation = CreateCompilation(source);
+
+            var sourceModule = compilation.SourceModule;
+            var sourceAssembly = (SourceAssemblySymbol)sourceModule.ContainingAssembly;
+
+            var retargetingAssembly = new RetargetingAssemblySymbol(sourceAssembly, isLinked: false);
```

Comment:
>var retargetingAssembly = new RetargetingAssemblySymbol(sourceAssembly, isLinked: false); [](start = 12, length = 89)

Please do not use this legacy way of testing retargeting. Create a scenario when the system itself creates one, a real scenario and test that.

Reply:
I've looked at more recent retargeting tests (Fred's recent PRs) but could not figure out the key to triggering retargeting logic to kick in. I tried tracing it back (seems to be triggered in ReferenceManager?).
What's a simple/modern test you'd recommend as a model for retargeting?

---
In reply to: [422534751](https://github.com/dotnet/roslyn/pull/44077#discussion_r422534751) [](ancestors = 422534751)

Reply:
> What's a simple/modern test you'd recommend as a model for retargeting?

For example, look at CodeGenTests.CustomFields_01. The 7th compilation, the commented out, is supposed to use retargeting symbols for comp1. The change in the target framework does the trick. 

---
In reply to: [422651853](https://github.com/dotnet/roslyn/pull/44077#discussion_r422651853) [](ancestors = 422651853,422534751)

Reply:
Thanks a lot. Changed the retargeting test to use that model.